lib/sneakers/worker.rb

```diff
@@ -61,11 +61,11 @@ def do_work(delivery_info, metadata, msg, handler)
           end
         rescue Timeout::Error
           res = :timeout
-          worker_error('timeout')
+          worker_error('timeout',msg)
         rescue => ex
           res = :error
           error = ex
-          worker_error('unexpected error', ex)
+          worker_error('unexpected error', msg, ex)
```

Comment:
We actually have been meaning to PR sneakers with some way to re-raise exceptions so that services like Sentry and Airbrake will pick them up. I think that would be better than logging every thing.